src/calculateRentalCost.js

```diff
@@ -4,7 +4,23 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const dailyRate = 40;
+  const longTermThreshold = 7;
+  const longTermDiscount = 50;
+  const shortTermThreshold = 3;
+  const shortTermDiscount = 20;
+
+  const totalBaseCost = days * dailyRate;
+
+  if (days >= longTermThreshold) {
+    return totalBaseCost - longTermDiscount;
+  }
+
+  if (days >= shortTermThreshold) {
+    return totalBaseCost - shortTermDiscount;
+  }
```

Comment:
You can simplify the conditional structure by removing the 'else' after 'if' with 'return'.